agenta-web/cypress/e2e/playground.cy.ts

```diff
@@ -0,0 +1,73 @@
+import {randString} from "../../src/lib/helpers/utils"
+
+describe("run a simple prompt", () => {
+    beforeEach(() => {
```

Comment:
we don't need beforeEach here as this is gonna be triggered in each test (before each it() )  use Before instead